src/main/java/sravanjava/SearchBus.java

```diff
@@ -0,0 +1,59 @@
+package sravanjava;
+
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+;
+
+public class SearchBus {
+    public static void main(String[] args) throws InterruptedException {
+
+//Logger creation
+        Logger logger= Logger.getLogger("SearchBus");
+
+//setting path & initializing chrome driver
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Sravan\\Selenium\\chromedriver.exe");
```

Comment:
this chrome driver is in your local folder, we will not be able to access this. please put the chromedriver.exe in framework